.env.example

```
@@ -9,6 +9,7 @@ API__ALLOWED_CORS_METHODS=["*"]
 AUTHENTICATION__ENABLED=true
 AUTHENTICATION__PUBLIC_KEY_PATH=./keys/jwt-key.pub
 AUTHENTICATION__JWT_ALGORITHM=RS256
+AUTHENTICATION__PRIVILEGED_ROLES=["admin"] # Will become more complex in future
```

```suggestion
AUTHENTICATION__PRIVILEGED_ROLES=["admin"]
```
I think its fine without the comment, unless there is any urgency on extra features it could stay like this for a long time.